src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/MethodsDropdown/style.ts

```diff
@@ -0,0 +1,50 @@
+import { makeStyles } from '@material-ui/core/styles'
+
+const buttonWidth = '140px'
```

Comment:
Can you not do this as a styled component? We aim not to do any new styling in MUI itself.

Reply:
This already existed and was extracted from `CurrencyDropdown/style` since both the `MethodsDropdown` and `CurrencyDropdown` are mui dropdowns but their trigger buttons look slightly different now.